packages/legacy/core/App/components/misc/CredentialCard11.tsx

```diff
@@ -119,6 +119,7 @@ const CredentialCard11: React.FC<CredentialCard11Props> = ({
     cardContainer: {
       flexDirection: 'row',
       minHeight: 0.33 * width,
+      paddingRight: 75,
```

Comment:
What is this large right padding for?

Reply:
so that long texts do not cut into the credential card
![Captura de tela de 2023-09-13 19-46-38](https://github.com/hyperledger/aries-mobile-agent-react-native/assets/69023582/3c2a520c-5895-4e41-a2d9-e39f24aac50a)

Reply:
Can we take the same approach to fix that as we did with the issuer name? Better for it to just wrap dynamically than to use an padding value that might not be responsive

Reply:
Which fix is that? Where can I find it?

Reply:
Lines 340-356 and lines 358-374 in this file (new version line numbers)

Reply:
it will be necessary to place a marginRight in this view because marginLeft is placing the view outside the main view and this only happens on the ProofRequest screen
![Captura de tela de 2023-09-20 00-24-46](https://github.com/hyperledger/aries-mobile-agent-react-native/assets/69023582/05ed0fd8-eede-4b8a-9fe6-104236cc4ab4) ![Captura de tela de 2023-09-20 00-24-08](https://github.com/hyperledger/aries-mobile-agent-react-native/assets/69023582/ca985021-4e05-4e01-8b20-6801efe0b292)